Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -650,6 +655,7 @@ void CGameListCtrl::SetBackgroundColor()
                                        wxSystemSettings::GetColour(wxSYS_COLOUR_WINDOW)) :
                                wxSystemSettings::GetColour(wxSYS_COLOUR_WINDOW);
     CGameListCtrl::SetItemBackgroundColour(i, color);
+    SetItemTextColour(i, contrast_text(color));
```

Comment:
It's a bit misleading that this is being done in a function called `SetBackgroundColor`. Maybe it should be renamed to something like `SetColors`?